src/services/walkthroughServices.js

```diff
@@ -87,6 +142,27 @@ const handleServiceInstanceWatchEvents = (dispatch, event) => {
       type: FULFILLED_ACTION(walkthroughTypes.CREATE_WALKTHROUGH),
       payload: event.payload
     });
+
+    const dashboardUrl = 'integreatly/dashboard-url';
```

Comment:
might be worth a comment in here explaining why we're doing this i.e. because the template broker doesn't have dashboard url support.
Is there an issue in origin to request this feature?

Reply:
Will do, there is an issue around not being able to add a dashboardURL to a template.

https://github.com/openshift/origin/issues/21082